src/DynamoRevit/DynamoRevit.cs

```diff
@@ -546,6 +562,27 @@ private static void TryOpenAndExecuteWorkspaceInCommandData(DynamoRevitCommandDa
                     dynamoViewModel.ShowStartPage = false;
                 }
 
+                //If we have information about the nodes and their values we want to push those values after the file is opened.
+                if (commandData.JournalData.ContainsKey(JournalKeys.ModelNodesInfo))
+                {
+                    try
+                    {
+                        var allNodesInfo = JsonConvert.DeserializeObject<List<Dictionary<string, string>>>(commandData.JournalData[JournalKeys.ModelNodesInfo]);
```

Comment:
We use this generic dictionary here which comes from a json string received as parameter instead of a more specific data structure that we taught we'll be harder to transmit as a parameter.